Edit primary branch in IDE release note and docs changes

matthewshaver opened the pull request:
## What are you changing in this pull request and why?

Release note describing the ability to edit primary branch in dbt Cloud IDE

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

[notion project](https://www.notion.so/dbtlabs/DevExp-Enhancements-CLI-IDE-Git-AI-9d72cc68bcc44a6bafe95eab793235fb)

matthewshaver committed:
Release note: edit primary branch in IDE

matthewshaver committed:
edit feture

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+sidebar_label: "New: Edit primary branch in IDE"
+sidebar_position: 06
+tags: [Feb-2024]
+date: 2024-02-21
```
```suggestion
date: 2024-02-22
```

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/read-only.png" width="90%" title="Old read-only experience"/>
+
+Now the branch is displayed with a lock icon to identify it as a protected branch:
```
```suggestion
The branch is now displayed with a lock icon to identify it as a protected branch:
```

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/protected.png" width="90%" title="New protected experience"/>
+
+When a developer makes a commit to the branch, they receive a prompt that warns them that it is a protected branch, with the option to cancel the commit or create a new branch:
```
```suggestion
When you a commit to the branch, you receive a prompt that warns them that it is a protected branch, with the option to cancel the commit or create a new branch:
```

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/commit-popup.png" width="90%" title="Protected branch pop-up window"/>
+
+When the user chooses **Create new branch**, they are presented with the window where they can enter the branch name and commit message:
```
```suggestion
When you choose **Create new branch**, you are presented with a window where you can enter the branch name and commit message:
```

matthewshaver committed:
Apply suggestions from code review

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+date: 2024-02-21
+---
+
+You can now edit the primary branch of your GitHub repo in the dbt Cloud IDE. The primary branch of GitHub has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits to be made. When a commit is made, dbt Cloud will prompt the developer to create a new branch. dbt Cloud will pre-populate the new branch name with the _username_-patch-#, how ...
```
```suggestion
You can now edit the primary branch of your GitHub repo in the dbt Cloud IDE. The primary branch of GitHub has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. debt Cloud will pre-populate the new branch name with the _username_-patch-#; however, you can edit the field with a custom branch name.
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/protected.png" width="90%" title="New protected experience"/>
+
+When you a commit to the branch, you receive a prompt that warns them that it is a protected branch, with the option to cancel the commit or create a new branch:
```
```suggestion
When you make a commit to the branch, you receive a prompt that warns that it is a protected branch, with the option to cancel the commit or create a new branch:
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md

naomiwritescode commented on website/static/img/docs/dbt-cloud/using-dbt-cloud/commit-popup.png:
We removed this modal at product/design's request

matthewshaver committed:
Updating screenshots

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,26 @@
+date: 2024-02-22
+---
+
+You can now edit the primary branch of your GitHub repo in the dbt Cloud IDE. The primary branch of GitHub has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. debt Cloud will pre-populate the new branch name with the _username_-patch-#; however, you can edit t ...
```
```suggestion
You can now edit the primary branch of your GitHub repo in the dbt Cloud IDE. The primary branch of GitHub has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. dbt Cloud will pre-populate the new branch name with the _username_-patch-#; however, you can edit the field with a custom branch name.
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md

matthewshaver committed:
Update ide-github.md

Editorial changes and removing a photo

mirnawong1 committed:
Merge branch 'current' into github-ide

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+---
+title: "New: Edit in primary branch in IDE"
+description: "February 2024: Edit access to the primary GitHub branch while in the IDE. This was previously a read-only feature."
```
hey @naomiwritescode is this only for github or for all git providers? if it's for all git providers, i think we can leave out github since we support other git providers. 

```suggestion
description: "February 2024: You can now edit directly on your primary git branch and commit those changes to a new branch when ready. This was previously a read-only feature."
```

matthewshaver replied:
Made a separate update

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+---
+title: "New: Edit in primary branch in IDE"
```
i dont mind new but is this an enhancement to an existing system?? it's an improved user experience 

```suggestion
title: "Enhancement: Edit in primary branch in IDE"
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+---
+title: "New: Edit in primary branch in IDE"
+description: "February 2024: Edit access to the primary GitHub branch while in the IDE. This was previously a read-only feature."
+sidebar_label: "New: Edit primary branch in IDE"
```
```suggestion
sidebar_label: "Enhancement: Edit primary branch in IDE"
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+date: 2024-02-22
+---
+
+You can now edit the primary branch of your GitHub repo in the dbt Cloud IDE. The primary branch of GitHub has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. dbt Cloud will pre-populate the new branch name with the USERNAME-patch-#; however, you can edit the  ...
```
i tested the managed myself myself and cried of joy -- @naomiwritescode am confirming this new change applies to managed repos as well as imported git URLs right? 🙏 

```suggestion
You can now edit, format, or lint files and execute dbt commands directly in your primary git branch within the dbt Cloud IDE.  This delightful enhancement is available across various types of repositories: native integrations, imported git URLs, and managed repos.

Previously, the primary git branch provided _read-only_ access in the IDE. This update changes the primary branch to _protected_, allowing you to edit as needed. When you're ready to commit your changes, dbt Cloud will prompt you to create a new branch and pre-populate the new branch name in the following format: `GIT_USERNAME-patch-#`. You can also edit the field with a custom branch name.
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/protected.png" width="90%" title="New protected experience"/>
+
+When you make a commit while on the primary branch, a model window will open prompting you to create a new branch and enter a commit message:
```
i think it's 'modal' for pop ups

```suggestion
When you make a commit while on the primary branch, a modal window will open prompting you to create a new branch and enter a commit message:
```

mirnawong1 reviewed (changes requested):
hey @matthewshaver , looks great and made some suggestions as i think we need to clarify if its github. specific or just git?

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+
+Previously, the primary branch was displayed as read-only:
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/read-only.png" width="90%" title="Old read-only experience"/>
```
```suggestion
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/read-only.png" width="90%" title="Previous read-only experience"/>
```

mirnawong1 commented:
also one more note, i wonder if we should use the `<DocCarousel slidesPerView={1}>` component to present the before and after. not sure if it's just me but the images look uneven and take up a lot of space. maybe the carousel will help keep it tidy.
![Screenshot 2024-02-23 at 09 42 45](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/988c1861-77a1-4ea9-8681-d50dc8f4647d)

```
<DocCarousel slidesPerView={1}>

<Lightbox src="/img/docs/...." width="100%" title="........"/>

<Lightbox src="/img/docs/...." width="100%" title="........"/>

<Lightbox src="/img/docs/...." width="100%" title="........"/>

</DocCarousel>
```

Commit:
update git changes

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-github.md:
```diff
@@ -0,0 +1,22 @@
+sidebar_label: "New: Edit primary branch in IDE"
+sidebar_position: 06
+tags: [Feb-2024]
+date: 2024-02-22
```
```suggestion
date: 2024-02-23
```

matthewshaver committed:
updating to reflect all git repos

Commit:
updates

mirnawong1 committed:
Merge branch 'github-ide' into mwong-add-git-enhancement

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,22 @@
+date: 2024-02-22
+---
+
+You can now edit the primary branch of your git repo in the dbt Cloud IDE. The primary branch of the connected git repo has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. dbt Cloud will pre-populate the new branch name with the USERNAME-patch-#; however, you  ...
```
```suggestion
You can now edit, format, or lint files and execute dbt commands directly in your primary git branch in the dbt Cloud IDE.  This enhancement is available across various repositories: native integrations, imported git URLs, and managed repos.

The primary branch of the connected git repo has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. dbt Cloud will pre-populate the new branch name with the USERNAME-patch-#; however, you can edit the field with a custom branch name.
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,24 @@
+---
+title: "New: Edit in primary git branch in IDE"
```
```suggestion
title: "Enhancement: Edit in primary git branch in IDE"
```

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,24 @@
+---
+title: "New: Edit in primary git branch in IDE"
+description: "February 2024: Edit access to the primary git branch while in the IDE. This was previously a read-only feature."
+sidebar_label: "New: Edit primary git branch in IDE"
```
```suggestion
sidebar_label: "Enhancement: Edit primary git branch in IDE"
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Commit:
update custom branch

matthewshaver committed:
Adding carousel

Commit:
last tweak

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+
+<DocCarousel slidesPerView={1}>
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/read-only.png" width="85%" title="Previous read-only experience"/>
```
```suggestion
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/read-only.png" width="75%" title="Previous read-only experience"/>
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/read-only.png" width="85%" title="Previous read-only experience"/>
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/protected.png" width="85%" title="New protected experience"/>
```
```suggestion
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/protected.png" width="75%" title="New protected experience"/>
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+
+When you make a commit while on the primary branch, a model window will open prompting you to create a new branch and enter a commit message:
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/create-new-branch.png" width="90%" title="Create new branch window"/>
```
```suggestion
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/create-new-branch.png" width="75%" title="Create new branch window"/>
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+---
+title: "Enhancement: Edit in primary git branch in IDE"
+description: "February 2024: Edit access to the primary git branch while in the IDE. This was previously a read-only feature."
```
i think we can clarify that they can commit to a new branch, just so it's super clear they can't commit changes in their main branch.

```suggestion
description: "February 2024: Edit directly on your primary git branch and commit those changes to a new branch when ready. This was previously a read-only feature." 
```

mirnawong1 replied:
wdyt?

mirnawong1 replied:
i want to make sure the description is super clear when it's announced in the slack community
<img width="920" alt="Screenshot 2024-02-23 at 15 17 31" src="https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/4aaf2c51-6f4b-4a8a-b39a-4e8e949429ea">

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+sidebar_label: "Enhancement: Edit primary git branch in IDE"
+sidebar_position: 06
+tags: [Feb-2024]
+date: 2024-02-22
```
```suggestion
date: 2024-02-23
```

mirnawong1 replied:
today's date

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+
+</DocCarousel>
+
+When you make a commit while on the primary branch, a model window will open prompting you to create a new branch and enter a commit message:
```
```suggestion
When you make a commit while on the primary branch, a modal window will open prompting you to create a new branch and enter a commit message:
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+date: 2024-02-22
+---
+
+You can now edit, format, or lint files and execute dbt commands directly in your primary git branch in the dbt Cloud IDE.  This enhancement is available across various repositories: native integrations, imported git URLs, and managed repos.
```
```suggestion
You can now edit, format, or lint files and execute dbt commands directly in your primary git branch in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  This enhancement is available across various repositories: native integrations, imported git URLs, and managed repos.
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+
+You can now edit, format, or lint files and execute dbt commands directly in your primary git branch in the dbt Cloud IDE.  This enhancement is available across various repositories: native integrations, imported git URLs, and managed repos.
+
+The primary branch of the connected git repo has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. dbt Cloud will pre-populate the new branch name with the USERNAME-patch-#; however, you can edit the field with a custom branch name.
```
```suggestion
The primary branch of the connected git repo has traditionally been _read-only_ in the IDE. This update changes the branch to _protected_ and allows direct edits. When a commit is made, dbt Cloud will prompt you to create a new branch. dbt Cloud will pre-populate the new branch name with the GIT_USERNAME-patch-#; however, you can edit the field with a custom branch name.
```

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

mirnawong1 committed:
Update ide-user-interface.md

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

mirnawong1 committed:
Update lint-format.md

mirnawong1 committed:
Update website/docs/docs/collaborate/git/version-control-basics.md

mirnawong1 committed:
Update website/docs/guides/bigquery-qs.md

mirnawong1 committed:
Update website/docs/guides/databricks-qs.md

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+---
+title: "Enhancement: Edit in primary git branch in IDE"
+description: "February 2024: Edit directly on your primary git branch and commit those changes to a new branch when ready. This was previously a read-only feature." 
```
```suggestion
description: "February 2024: Edit directly on your primary git branch and create a new branch when ready to commit. The primary branch was previously read-only but now allows editing while remaining protected." 
```

mirnawong1 replied:
this reads lovely!

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

mirnawong1 committed:
Update website/docs/guides/microsoft-fabric-qs.md

mirnawong1 committed:
Update website/docs/guides/redshift-qs.md

mirnawong1 committed:
Update website/docs/guides/snowflake-qs.md

mirnawong1 committed:
Update website/docs/guides/starburst-galaxy-qs.md

matthewshaver commented on website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md:
```diff
@@ -0,0 +1,26 @@
+date: 2024-02-23
+---
+
+You can now edit, format, or lint files and execute dbt commands directly in your primary git branch in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  This enhancement is available across various repositories: native integrations, imported git URLs, and managed repos.
```
```suggestion
You can now edit, format, or lint files and execute dbt commands directly in your primary git branch in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  This enhancement is available across various repositories, including native integrations, imported git URLs, and managed repos.
```

mirnawong1 committed:
Merge branch 'github-ide' into mwong-add-git-enhancement

matthewshaver committed:
Update website/docs/docs/dbt-versions/release-notes/72-Feb-2024/ide-git-edit.md

mirnawong1 committed:
Merge branch 'github-ide' into mwong-add-git-enhancement

Commit:
tweak

Commit:
Merge branch 'mwong-add-git-enhancement' of https://github.com/dbt-labs/docs.getdbt.com into mwong-add-git-enhancement

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/faqs/Environments/custom-branch-settings.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/faqs/Environments/custom-branch-settings.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/bigquery-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/bigquery-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/snippets/quickstarts/schedule-a-job.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/bigquery-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/bigquery-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/databricks-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/databricks-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/databricks-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/snippets/quickstarts/schedule-a-job.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/snippets/quickstarts/schedule-a-job.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/dbt-cloud-ide/ide-user-interface.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/guides/databricks-qs.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

Commit:
fold feedback

mirnawong1 committed:
update docs to reflect main git enhancements (#4980)

this pr updates the following pages to account for the new and improved
main branch workflow in the ide:

- IDE git button: https://arc.net/l/quote/kgbayfdq
- IDE UI: https://arc.net/l/quote/edletcth
- IDE linting:
https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/lint-format#format-sql
- quickstart workflow: https://arc.net/l/quote/okqvcakv
- possibly this faq;
https://docs.getdbt.com/faqs/Environments/custom-branch-settings#development

[notion
project](https://www.notion.so/dbtlabs/DevExp-Enhancements-CLI-IDE-Git-AI-9d72cc68bcc44a6bafe95eab793235fb)

mirnawong1 committed:
Merge branch 'current' into github-ide

mirnawong1 committed:
Merge branch 'current' into github-ide

Commit:
update fo consistency

mirnawong1 reviewed (commented):
lgtm!

matthewshaver committed:
Merge branch 'current' into github-ide
